src/lib/JoinRoom.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it, expect, vi } from 'vitest';
+import { render, fireEvent, screen, logDOM } from '@testing-library/svelte';
+import '@testing-library/jest-dom';
+import JoinRoom from './JoinRoom.svelte';
+
+describe('JoinRoom', () => {
+  
+	it('should accept any pattern if none passed in, enable button', async () => {
+		render(JoinRoom);
```

Comment:
nit:

Seems like you've got some funky indentation stuff going on with your editor or something maybe?